src/renderer/components/watch-video-comments/watch-video-comments.vue

```diff
@@ -48,6 +48,7 @@
         <img
           :src="comment.authorThumb"
           class="commentThumbnail"
```

Comment:
Could you please add `cursor: pointer` to the CSS of this class? We want users to know that this is clickable.